XiEditor/Search.swift

```diff
@@ -121,7 +121,6 @@ extension EditViewController {
     func find(_ term: String?, case_sensitive: Bool, wrap_around: Bool) {
```

Comment:
Yes, good find. Will be updated with the commit that address all other comments. Was a quick commit to provide compatibility for the xi-editor PR